README.md

```diff
@@ -35,14 +63,6 @@ Customer environment is expected to be based on RHEL distributions. By default t
 
 Task specific:
 
-- `AWS_ACCESS_KEY_ID` - AWS access key id for AWS deployment.
```

Comment:
Why remove these variables , they are needed when deploying OA outside of CI... we should also update the instructions on OA for oa_deployment_type instead of deployment_type , these variables have different purposes.

Reply:
There is a note in description, which says that you need to setup environment variables from `List of other environment variables` before, and there is the `AWS_ACCESS_KEY_ID` present.